command/vault/secrets_push.go

```diff
@@ -32,7 +32,7 @@ func SecretsPushWithConfig(c *cli.Context, config *config.Config) error {
 
```

Comment:
this is a breaking, unrelated change to your PR title, please open a new PR for discussion on this